src/js/utils/Util.js

```diff
@@ -1,5 +1,26 @@
 const Util = {
 
+  findWhere: function (findInObject, attributes) {
+    let keys = Object.keys(attributes);
+    let length = keys.length;
+
+    return Array.find.call(null, findInObject, function (object) {
```

Comment:
why don't we just do `findInObject.find(`?

can we call `findInObject` something else because it's an array not an object?